integration/tests/basic/test_rpc_calls.py

```diff
@@ -452,6 +452,29 @@ def test_rpc_call_eth_send_raw_transaction(self):
             response["result"]
         ), f"Invalid response result {response['result']}"
 
+    def test_max_transaction_size(self):
```

Comment:
pls rename to:
test_eth_sendRawTransaction_max_size
test_eth_sendRawTransaction_max_contract_size

Reply:
Sure, done